src/patternfly/layouts/Grid/grid.scss

```diff
@@ -1,5 +1,3 @@
-$pf-l-grid--breakpoint-map: build-breakpoint-map();
```

Comment:
looks like this is still removed - did you mean to add it back in?

Reply:
whoops, updated this